docs/self-managed/identity/user-guide/configuration/configure-external-identity-provider.md

```diff
@@ -32,7 +32,9 @@ To configure an external identity provider like OpenID Connect, SAML, LDAP, or A
       ![keycloak-add-user-federation](../img/keycloak-add-user-federation.png)
 
 :::tip
-Keycloak supports a wide variety of authentication options, such as mapping external user groups, roles, or scopes to internal roles, and configuring the login screen and flow when multiple providers are added. Visit the Keycloak documentation for details on [adding a provider](https://www.keycloak.org/docs/19.0.3/server_admin/index.html#adding-a-provider),
+Keycloak supports a wide variety of authentication options, such as mapping external user groups, roles, or scopes to internal roles, and configuring the login screen and flow when multiple providers are added.
+
+Visit the Keycloak documentation for details on [adding a provider](https://www.keycloak.org/docs/19.0.3/server_admin/index.html#adding-a-provider),
 [configuring authentication](https://www.keycloak.org/docs/19.0.3/server_admin/index.html#configuring-authentication), and
 [integrating identity providers](https://www.keycloak.org/docs/19.0.3/server_admin/index.html#_identity_broker).
 :::
```

Comment:
Based on the table in [Supported environments in 8.2](https://docs.camunda.io/docs/reference/supported-environments/#camunda-platform-8-self-managed), we support Keycloak 16.1.x, 18.x, and 19.x. Pointing to the most recent version we support makes sense.

However, in[ `/next/`](https://docs.camunda.io/docs/next/reference/supported-environments/#camunda-platform-8-self-managed) where these docs changes are happening in this PR, we mention supported for 21.x only. These links should be adjusted.

I don't see a great way to send people to Keycloak docs without a specific version in the URL. We could consider some variable per C8 version, but that would be out of scope of this PR.

Reply:
Other URLs in this PR would need to be adjusted too.

Reply:
For clarity, we also support Keycloak v21 in the 8.2 branch now however this needs to be added to the matrix. I have created a separate issue for this https://github.com/camunda-cloud/identity/issues/1976

Reply:
@Ben-Sheppard, for something like this where the direction is clear (adding v21 to 8.2), we can update the matrix and URLs, then add you as a reviewer. Negotiate with us! We are happy to help as we can.